I 03-17 21:25:52 controller.py:59] DAG:
I 03-17 21:25:52 controller.py:59] [Task<name=compute-text-vectors>(run='# Initialize and dow...')
I 03-17 21:25:52 controller.py:59]   resources: {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}]
I 03-17 21:25:52 controller.py:195] Submitted managed job 471 (task: 0, name: 'compute-text-vectors'); SKYPILOT_TASK_ID: sky-managed-2025-03-17-21-25-52-860119_compute-text-vectors_471-0
I 03-17 21:25:52 controller.py:199] Started monitoring.
I 03-17 21:25:52 state.py:451] Launching the spot cluster...
I 03-17 21:25:53 storage.py:1058] Storage type StoreType.S3 already exists.
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 21:25:54 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:25:54 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:25:54 optimizer.py:749] 
I 03-17 21:25:54 optimizer.py:884] Considered resources (1 node):
I 03-17 21:25:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:25:54 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:25:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:25:54 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1b   0.22       [32m   âœ”[0m     
I 03-17 21:25:54 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:25:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:25:54 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1b).
[2mW 03-17 21:40:23 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1a.[0m
W 03-17 21:40:24 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:40:24 cloud_vm_ray_backend.py:2097] 
W 03-17 21:40:24 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:40:24 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:40:24 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:40:24 optimizer.py:749] 
I 03-17 21:40:24 optimizer.py:884] Considered resources (1 node):
I 03-17 21:40:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:40:24 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:40:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:40:24 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1a   0.25       [32m   âœ”[0m     
I 03-17 21:40:24 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:40:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:40:25 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a).
[2mW 03-17 21:41:11 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:41:12 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:41:12 cloud_vm_ray_backend.py:2097] 
W 03-17 21:41:12 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:41:12 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:41:12 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:41:12 optimizer.py:749] 
I 03-17 21:41:12 optimizer.py:884] Considered resources (1 node):
I 03-17 21:41:12 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:41:12 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:41:12 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:41:12 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2d   0.25       [32m   âœ”[0m     
I 03-17 21:41:12 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:41:12 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:41:12 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2d).
[2mW 03-17 21:45:32 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:45:33 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:45:33 cloud_vm_ray_backend.py:2097] 
W 03-17 21:45:33 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:45:34 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:45:34 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:45:34 optimizer.py:749] 
I 03-17 21:45:34 optimizer.py:884] Considered resources (1 node):
I 03-17 21:45:34 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:45:34 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:45:34 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:45:34 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2a   0.27       [32m   âœ”[0m     
I 03-17 21:45:34 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:45:34 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:45:34 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a).
W 03-17 21:58:43 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:58:43 cloud_vm_ray_backend.py:2097] 
W 03-17 21:58:43 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:58:43 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:58:43 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:58:43 optimizer.py:749] 
I 03-17 21:58:43 optimizer.py:884] Considered resources (1 node):
I 03-17 21:58:43 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:58:43 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:58:43 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:58:43 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1b    0.30       [32m   âœ”[0m     
I 03-17 21:58:43 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:58:43 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:58:44 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1b).
[2mW 03-17 22:06:34 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-1a, us-east-1c, us-east-1d.[0m
W 03-17 22:06:35 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:06:35 cloud_vm_ray_backend.py:2097] 
W 03-17 22:06:35 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:06:35 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:06:35 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:06:35 optimizer.py:749] 
I 03-17 22:06:35 optimizer.py:884] Considered resources (1 node):
I 03-17 22:06:35 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:06:35 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:06:35 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:06:35 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2c    0.31       [32m   âœ”[0m     
I 03-17 22:06:35 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:06:35 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:06:35 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2c).
[2mW 03-17 22:08:06 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a, us-east-2b.[0m
W 03-17 22:08:07 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:08:07 cloud_vm_ray_backend.py:2097] 
W 03-17 22:08:07 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:08:07 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:08:07 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:08:07 optimizer.py:749] 
I 03-17 22:08:07 optimizer.py:884] Considered resources (1 node):
I 03-17 22:08:07 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:08:07 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:08:07 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:08:07 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2b    0.32       [32m   âœ”[0m     
I 03-17 22:08:07 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:08:07 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:08:07 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2b).
[2mW 03-17 22:16:10 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:16:11 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:11 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:11 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:11 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:11 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:16:11 optimizer.py:749] 
I 03-17 22:16:11 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:11 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:16:11 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:16:11 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:16:11 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1a   0.33       [32m   âœ”[0m     
I 03-17 22:16:11 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:16:11 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:11 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a).
[2mW 03-17 22:16:16 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:16:17 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:17 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:17 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:17 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:17 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:16:17 optimizer.py:749] 
I 03-17 22:16:17 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:17 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:16:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:17 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1c    0.34       [32m   âœ”[0m     
I 03-17 22:16:17 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:16:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:17 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1c).
[2mE 03-17 22:16:18 config.py:46] SKYPILOT_ERROR_NO_NODES_LAUNCHED: No usable subnets matching availability zone ['us-east-1c'] found. Choose a different availability zone or try manually creating an instance in your specified region to populate the list of subnets and trying this again. If you have set `use_internal_ips`, check that this zone has a subnet that (1) has the substring 'private' in its name tag and (2) does not assign public IPs (`map_public_ip_on_launch` is False).[0m
W 03-17 22:16:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:19 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:16:19 optimizer.py:749] 
I 03-17 22:16:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:19 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:16:19 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:16:19 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:16:19 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1b   0.34       [32m   âœ”[0m     
I 03-17 22:16:19 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:16:19 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:19 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1b).
[2mW 03-17 22:16:24 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:16:24 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:24 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:24 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:25 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:25 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:16:25 optimizer.py:749] 
I 03-17 22:16:25 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:25 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:25 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:16:25 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:25 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2a    0.34       [32m   âœ”[0m     
I 03-17 22:16:25 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:16:25 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:25 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a).
[2mW 03-17 22:16:36 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:16:37 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:37 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:37 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:37 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:37 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:16:37 optimizer.py:749] 
I 03-17 22:16:37 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:37 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:16:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:37 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3c    0.35       [32m   âœ”[0m     
I 03-17 22:16:37 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:16:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:37 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3c).
[2mW 03-17 22:16:45 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:16:46 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:46 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:46 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:46 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:46 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:16:46 optimizer.py:749] 
I 03-17 22:16:46 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:46 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:46 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:16:46 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:46 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1a   0.35       [32m   âœ”[0m     
I 03-17 22:16:46 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:16:46 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:46 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a).
[2mW 03-17 22:16:58 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:16:58 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:58 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:58 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:58 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:58 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:16:58 optimizer.py:749] 
I 03-17 22:16:58 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:58 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:58 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:16:58 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:58 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2b    0.36       [32m   âœ”[0m     
I 03-17 22:16:58 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:16:58 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:59 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2b).
[2mW 03-17 22:17:06 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:17:07 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:17:07 cloud_vm_ray_backend.py:2097] 
W 03-17 22:17:07 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:17:07 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:17:07 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:17:07 optimizer.py:749] 
I 03-17 22:17:07 optimizer.py:884] Considered resources (1 node):
I 03-17 22:17:07 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:17:07 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:17:07 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:17:07 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2c   0.36       [32m   âœ”[0m     
I 03-17 22:17:07 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:17:07 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:17:07 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2c).
[2mW 03-17 22:17:18 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-southeast-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-southeast-2a.[0m
W 03-17 22:17:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:17:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:17:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:17:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:17:19 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:17:19 optimizer.py:749] 
I 03-17 22:17:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:17:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:17:19 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:17:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:17:19 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1d    0.37       [32m   âœ”[0m     
I 03-17 22:17:19 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:17:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:17:20 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1d).
[2mW 03-17 22:17:32 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:17:33 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:17:33 cloud_vm_ray_backend.py:2097] 
W 03-17 22:17:33 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:17:33 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:17:33 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:17:33 optimizer.py:749] 
I 03-17 22:17:33 optimizer.py:884] Considered resources (1 node):
I 03-17 22:17:33 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:17:33 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:17:33 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:17:33 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2b    0.37       [32m   âœ”[0m     
I 03-17 22:17:33 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:17:33 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:17:33 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2b).
[2mW 03-17 22:18:12 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-west-2b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-2a, us-west-2c, us-west-2d.[0m
W 03-17 22:18:13 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:13 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:13 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:13 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:13 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:18:13 optimizer.py:749] 
I 03-17 22:18:13 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:13 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:18:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:13 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1b   0.37       [32m   âœ”[0m     
I 03-17 22:18:13 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:18:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:14 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1b).
[2mW 03-17 22:18:24 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-north-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-north-1a.[0m
W 03-17 22:18:25 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:25 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:25 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:25 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:25 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:18:25 optimizer.py:749] 
I 03-17 22:18:25 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:25 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:18:25 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:18:25 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:18:25 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2a   0.38       [32m   âœ”[0m     
I 03-17 22:18:25 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:18:25 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:25 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2a).
[2mW 03-17 22:18:35 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-southeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-southeast-2c.[0m
W 03-17 22:18:37 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:37 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:37 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:37 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:37 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:18:37 optimizer.py:749] 
I 03-17 22:18:37 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:37 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:18:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:37 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3b    0.38       [32m   âœ”[0m     
I 03-17 22:18:37 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:18:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:37 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3b).
[2mW 03-17 22:18:44 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-west-3b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-west-3c.[0m
W 03-17 22:18:45 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:45 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:45 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:45 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:45 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:18:45 optimizer.py:749] 
I 03-17 22:18:45 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:45 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:45 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:18:45 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:45 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1a    0.39       [32m   âœ”[0m     
I 03-17 22:18:45 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:18:45 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:45 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a).
[2mW 03-17 22:18:59 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:18:59 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:59 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:59 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:59 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:59 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:18:59 optimizer.py:749] 
I 03-17 22:18:59 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:59 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:59 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:18:59 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:59 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2a    0.40       [32m   âœ”[0m     
I 03-17 22:18:59 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:18:59 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:59 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a).
[2mW 03-17 22:24:20 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2b, us-east-2c.[0m
W 03-17 22:24:21 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:24:21 cloud_vm_ray_backend.py:2097] 
W 03-17 22:24:21 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:24:21 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:24:21 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:24:21 optimizer.py:749] 
I 03-17 22:24:21 optimizer.py:884] Considered resources (1 node):
I 03-17 22:24:21 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:24:21 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:24:21 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:24:21 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-central-1b   0.40       [32m   âœ”[0m     
I 03-17 22:24:21 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:24:21 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:24:21 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1b).
[2mW 03-17 22:25:17 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:25:17 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-central-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:25:17 cloud_vm_ray_backend.py:2097] 
W 03-17 22:25:17 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:25:17 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:25:17 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:25:17 optimizer.py:749] 
I 03-17 22:25:17 optimizer.py:884] Considered resources (1 node):
I 03-17 22:25:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:25:17 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:25:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:25:17 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2d    0.41       [32m   âœ”[0m     
I 03-17 22:25:17 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:25:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:25:18 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2d).
[2mW 03-17 22:26:51 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:26:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:26:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:26:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:26:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:26:52 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:26:52 optimizer.py:749] 
I 03-17 22:26:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:26:52 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:26:52 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:26:52 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:26:52 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-central-1a   0.41       [32m   âœ”[0m     
I 03-17 22:26:52 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:26:52 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:26:52 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1a).
[2mW 03-17 22:27:01 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:27:02 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-central-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:27:02 cloud_vm_ray_backend.py:2097] 
W 03-17 22:27:02 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:27:03 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:27:03 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:27:03 optimizer.py:749] 
I 03-17 22:27:03 optimizer.py:884] Considered resources (1 node):
I 03-17 22:27:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:27:03 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:27:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:27:03 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2c    0.41       [32m   âœ”[0m     
I 03-17 22:27:03 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:27:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:27:03 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2c).
[2mW 03-17 22:28:02 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:28:02 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:28:02 cloud_vm_ray_backend.py:2097] 
W 03-17 22:28:02 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:28:02 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:28:02 optimizer.py:749] [1mEstimated cost: [0m$0.5 / hour
I 03-17 22:28:02 optimizer.py:749] 
I 03-17 22:28:02 optimizer.py:884] Considered resources (1 node):
I 03-17 22:28:02 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:28:02 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:28:02 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:28:02 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-1a   0.46       [32m   âœ”[0m     
I 03-17 22:28:02 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:28:02 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:28:03 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-1[0m (ap-northeast-1a).
[2mW 03-17 22:28:13 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-1a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-1c.[0m
W 03-17 22:28:14 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:28:14 cloud_vm_ray_backend.py:2097] 
W 03-17 22:28:14 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:28:15 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:28:15 optimizer.py:749] [1mEstimated cost: [0m$0.5 / hour
I 03-17 22:28:15 optimizer.py:749] 
I 03-17 22:28:15 optimizer.py:884] Considered resources (1 node):
I 03-17 22:28:15 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:28:15 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:28:15 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:28:15 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-1c   0.48       [32m   âœ”[0m     
I 03-17 22:28:15 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:28:15 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:28:15 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-1[0m (ap-northeast-1c).
[2mW 03-17 22:28:26 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:28:27 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-1c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:28:27 cloud_vm_ray_backend.py:2097] 
W 03-17 22:28:27 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:28:27 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:28:27 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 22:28:27 optimizer.py:749] 
I 03-17 22:28:27 optimizer.py:884] Considered resources (1 node):
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:28:27 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:28:27 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-1     0.80       [32m   âœ”[0m     
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:28:27 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:28:27 cloud_vm_ray_backend.py:2097] 
W 03-17 22:28:27 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:28:27 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:28:27 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 22:28:27 optimizer.py:749] 
I 03-17 22:28:27 optimizer.py:884] Considered resources (1 node):
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:28:27 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:28:27 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-2     0.80       [32m   âœ”[0m     
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:28:27 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:28:27 cloud_vm_ray_backend.py:2097] 
W 03-17 22:28:27 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:28:27 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:28:27 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 22:28:27 optimizer.py:749] 
I 03-17 22:28:27 optimizer.py:884] Considered resources (1 node):
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:28:27 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:28:27 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-west-2     0.80       [32m   âœ”[0m     
I 03-17 22:28:27 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:28:27 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2a).
[2mW 03-17 22:32:18 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-west-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-2b, us-west-2c, us-west-2d.[0m
W 03-17 22:32:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-west-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:19 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 22:32:19 optimizer.py:749] 
I 03-17 22:32:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-north-1    0.85       [32m   âœ”[0m     
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-north-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:19 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 22:32:19 optimizer.py:749] 
I 03-17 22:32:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:19 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 22:32:19 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ca-central-1   0.89       [32m   âœ”[0m     
I 03-17 22:32:19 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ca-central-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:19 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:32:19 optimizer.py:749] 
I 03-17 22:32:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-south-1    0.97       [32m   âœ”[0m     
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-south-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:19 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:32:19 optimizer.py:749] 
I 03-17 22:32:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:19 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 22:32:19 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-northeast-2   0.99       [32m   âœ”[0m     
I 03-17 22:32:19 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-northeast-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:19 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:32:19 optimizer.py:749] 
I 03-17 22:32:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:19 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 22:32:19 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-central-1   1.01       [32m   âœ”[0m     
I 03-17 22:32:19 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-central-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:19 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:32:19 optimizer.py:749] 
I 03-17 22:32:19 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:19 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-2     1.02       [32m   âœ”[0m     
I 03-17 22:32:19 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-west-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:20 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:32:20 optimizer.py:749] 
I 03-17 22:32:20 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:20 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:20 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:32:20 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:32:20 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-3     1.02       [32m   âœ”[0m     
I 03-17 22:32:20 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-west-3 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:20 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:32:20 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:32:20 optimizer.py:749] 
I 03-17 22:32:20 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:20 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:32:20 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 22:32:20 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:32:20 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-southeast-2   1.05       [32m   âœ”[0m     
I 03-17 22:32:20 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-southeast-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:20 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:32:20 optimizer.py:749] [1mEstimated cost: [0m$1.2 / hour
I 03-17 22:32:20 optimizer.py:749] 
I 03-17 22:32:20 optimizer.py:884] Considered resources (1 node):
I 03-17 22:32:20 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:32:20 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 22:32:20 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:32:20 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-northeast-1   1.17       [32m   âœ”[0m     
I 03-17 22:32:20 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:32:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-northeast-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:32:20 cloud_vm_ray_backend.py:2097] 
W 03-17 22:32:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log\u001b[0m</rich_update>"</sky-payload>
E 03-17 22:32:20 cloud_vm_ray_backend.py:2904] [0m[31mâ¨¯[0m Failed to provision resources. [2mView logs: sky api logs -l sky-2025-03-17-21-25-53-486834/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 22:32:20 recovery_strategy.py:326] Failed to launch a cluster with error: sky.exceptions.ResourcesUnavailableError: [31mFailed to provision all possible launchable resources.[0m Relax the task's resource requirements: 1x AWS([Spot], cpus=4, {'L4': 1})
I 03-17 22:32:20 recovery_strategy.py:326] To keep retrying until the cluster is up, use the `--retry-until-up` flag.
I 03-17 22:32:20 recovery_strategy.py:326] Reasons for provision failures (for details, please check the log above):
I 03-17 22:32:20 recovery_strategy.py:326] Resource         Reason                                                      
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-south-1b for              
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-south-1a for              
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-2d for          
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-2a for          
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1b for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-2c for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-2b for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ca-central-1a for            
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1c for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ca-central-1b for            
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-2a for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-3c for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-north-1a for              
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-2b for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-southeast-2c for          
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1d for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-west-2b for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-north-1b for              
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-southeast-2a for          
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-3b for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1a for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-2a for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-central-1b for            
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-west-2d for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-central-1a for            
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-west-2c for {AWS([Spot],  
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-1a for          
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-1c for          
I 03-17 22:32:20 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in us-east-1 for   
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in us-east-2 for   
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in us-west-2 for   
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-north-1 for  
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ca-central-1    
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:32:20 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-south-1 for  
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-northeast-2  
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:32:20 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-central-1    
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:32:20 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-west-2 for   
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-west-3 for   
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-southeast-2  
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:32:20 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:32:20 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-northeast-1  
I 03-17 22:32:20 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:32:20 recovery_strategy.py:326]                  1})}.                                                       )
I 03-17 22:32:20 recovery_strategy.py:364] Retrying to launch the cluster in 62.9 seconds.
I 03-17 22:33:23 storage.py:699] Verifying bucket for storage skypilot-filemounts-root-726f1974-enw4397h
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 22:33:24 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:33:24 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 22:33:24 optimizer.py:749] 
I 03-17 22:33:24 optimizer.py:884] Considered resources (1 node):
I 03-17 22:33:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:33:24 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:33:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:33:24 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1b   0.22       [32m   âœ”[0m     
I 03-17 22:33:24 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:33:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:33:24 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1b).
[2mW 03-17 22:33:37 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:33:38 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:33:38 cloud_vm_ray_backend.py:2097] 
W 03-17 22:33:38 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:33:38 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:33:38 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 22:33:38 optimizer.py:749] 
I 03-17 22:33:38 optimizer.py:884] Considered resources (1 node):
I 03-17 22:33:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:33:38 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:33:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:33:38 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1a   0.25       [32m   âœ”[0m     
I 03-17 22:33:38 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:33:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:33:38 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a).
[2mW 03-17 22:33:51 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:33:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:33:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:33:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:33:53 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:33:53 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:33:53 optimizer.py:749] 
I 03-17 22:33:53 optimizer.py:884] Considered resources (1 node):
I 03-17 22:33:53 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:33:53 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:33:53 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:33:53 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2d   0.25       [32m   âœ”[0m     
I 03-17 22:33:53 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:33:53 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:33:53 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2d).
[2mW 03-17 22:34:06 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:34:08 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:34:08 cloud_vm_ray_backend.py:2097] 
W 03-17 22:34:08 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:34:08 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:34:08 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:34:08 optimizer.py:749] 
I 03-17 22:34:08 optimizer.py:884] Considered resources (1 node):
I 03-17 22:34:08 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:34:08 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:34:08 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:34:08 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2a   0.27       [32m   âœ”[0m     
I 03-17 22:34:08 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:34:08 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:34:08 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a).
[2mW 03-17 22:34:20 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2d.[0m
W 03-17 22:34:21 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:34:21 cloud_vm_ray_backend.py:2097] 
W 03-17 22:34:21 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:34:22 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:34:22 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:34:22 optimizer.py:749] 
I 03-17 22:34:22 optimizer.py:884] Considered resources (1 node):
I 03-17 22:34:22 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:22 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:34:22 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:22 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1b    0.30       [32m   âœ”[0m     
I 03-17 22:34:22 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:34:22 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:34:22 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1b).
[2mW 03-17 22:34:37 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:34:37 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:34:37 cloud_vm_ray_backend.py:2097] 
W 03-17 22:34:37 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:34:37 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:34:37 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:34:37 optimizer.py:749] 
I 03-17 22:34:37 optimizer.py:884] Considered resources (1 node):
I 03-17 22:34:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:37 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:34:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:37 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2c    0.31       [32m   âœ”[0m     
I 03-17 22:34:37 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:34:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:34:37 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2c).
[2mW 03-17 22:34:44 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a, us-east-2b.[0m
W 03-17 22:34:44 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:34:44 cloud_vm_ray_backend.py:2097] 
W 03-17 22:34:44 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:34:45 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:34:45 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:34:45 optimizer.py:749] 
I 03-17 22:34:45 optimizer.py:884] Considered resources (1 node):
I 03-17 22:34:45 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:45 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:34:45 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:45 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2b    0.32       [32m   âœ”[0m     
I 03-17 22:34:45 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:34:45 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:34:45 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2b).
[2mW 03-17 22:34:52 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:34:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:34:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:34:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:34:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:34:52 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:34:52 optimizer.py:749] 
I 03-17 22:34:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:34:52 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:34:52 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:34:52 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:34:52 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1a   0.33       [32m   âœ”[0m     
I 03-17 22:34:52 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:34:52 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:34:52 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a).
[2mW 03-17 22:34:57 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:34:58 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:34:58 cloud_vm_ray_backend.py:2097] 
W 03-17 22:34:58 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:34:58 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:34:58 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:34:58 optimizer.py:749] 
I 03-17 22:34:58 optimizer.py:884] Considered resources (1 node):
I 03-17 22:34:58 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:58 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:34:58 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:34:58 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1c    0.34       [32m   âœ”[0m     
I 03-17 22:34:58 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:34:58 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:34:58 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1c).
[2mE 03-17 22:34:59 config.py:46] SKYPILOT_ERROR_NO_NODES_LAUNCHED: No usable subnets matching availability zone ['us-east-1c'] found. Choose a different availability zone or try manually creating an instance in your specified region to populate the list of subnets and trying this again. If you have set `use_internal_ips`, check that this zone has a subnet that (1) has the substring 'private' in its name tag and (2) does not assign public IPs (`map_public_ip_on_launch` is False).[0m
W 03-17 22:35:00 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:35:00 cloud_vm_ray_backend.py:2097] 
W 03-17 22:35:00 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:35:00 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:35:00 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:35:00 optimizer.py:749] 
I 03-17 22:35:00 optimizer.py:884] Considered resources (1 node):
I 03-17 22:35:00 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:35:00 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:35:00 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:35:00 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1b   0.34       [32m   âœ”[0m     
I 03-17 22:35:00 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:35:00 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:35:00 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1b).
[2mW 03-17 22:35:05 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:35:05 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:35:05 cloud_vm_ray_backend.py:2097] 
W 03-17 22:35:05 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:35:06 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:35:06 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:35:06 optimizer.py:749] 
I 03-17 22:35:06 optimizer.py:884] Considered resources (1 node):
I 03-17 22:35:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:35:06 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:35:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:35:06 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2a    0.34       [32m   âœ”[0m     
I 03-17 22:35:06 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:35:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:35:06 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a).
[2mW 03-17 22:35:13 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:35:14 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:35:14 cloud_vm_ray_backend.py:2097] 
W 03-17 22:35:14 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:35:14 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:35:14 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:35:14 optimizer.py:749] 
I 03-17 22:35:14 optimizer.py:884] Considered resources (1 node):
I 03-17 22:35:14 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:35:14 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:35:14 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:35:14 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3c    0.35       [32m   âœ”[0m     
I 03-17 22:35:14 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:35:14 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:35:15 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3c).
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Waiting for SSH access[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:35:38 provisioner.py:451] [2mâ””â”€â”€ [0m[2mInstance is up.[0m
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (1/3 - initializing)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (2/3 - dependencies)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (3/3 - runtime)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:36:05 provisioner.py:632] [0m[32mâœ“ Cluster launched: compute-text-vectors-471.[0m  [2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 22:36:05 execution.py:332] [0mâš™ï¸Ž Syncing files.
<sky-payload>"<rich_init>[bold cyan]Syncing files[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing files[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Syncing files[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing file mounts[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/file_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:36:05 backend_utils.py:1315]   [2mSyncing (to 1 node): s3://skypilot-filemounts-root-726f1974-enw4397h/job-enw4397h/workdir -> ~/sky_workdir[0m
I 03-17 22:36:08 cloud_vm_ray_backend.py:4801] [0m[32mâœ“ Synced file_mounts.[0m  [2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/file_mounts.log[0m
<sky-payload>"<rich_update>[bold cyan]Mounting 1 storage[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/storage_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:36:08 backend_utils.py:1315]   [2mMounting (to 1 node): s3://sky-text-embeddings-multi-cluster-final-2 -> /output[0m
I 03-17 22:36:11 cloud_vm_ray_backend.py:4892] [0m[32mâœ“ Storage mounted.[0m  [2mView logs: sky api logs -l sky-2025-03-17-22-33-23-907400/storage_mounts.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Running setup[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Running setup[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Running setup[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
I 03-17 22:36:12 cloud_vm_ray_backend.py:3357] [0m[32mâœ“ Setup detached.[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Submitting job[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Submitting job[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Submitting job[/]</rich_start>"</sky-payload>
I 03-17 22:36:13 cloud_vm_ray_backend.py:3546] Multiple resources are specified for the task, using: AWS([Spot], cpus=4, {'L4': 1})
I 03-17 22:36:15 cloud_vm_ray_backend.py:3460] [0mâš™ï¸Ž Job submitted, ID: 1
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 22:36:15 recovery_strategy.py:289] Managed job cluster launched.
I 03-17 22:36:19 utils.py:131] === Checking the job status... ===
I 03-17 22:36:19 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:36:19 utils.py:140] ==================================
I 03-17 22:36:20 state.py:471] Job started.
I 03-17 22:36:40 utils.py:131] === Checking the job status... ===
I 03-17 22:36:41 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:36:41 utils.py:140] ==================================
I 03-17 22:37:01 utils.py:131] === Checking the job status... ===
I 03-17 22:37:02 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:37:02 utils.py:140] ==================================
I 03-17 22:37:22 utils.py:131] === Checking the job status... ===
I 03-17 22:37:23 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:37:23 utils.py:140] ==================================
I 03-17 22:37:43 utils.py:131] === Checking the job status... ===
I 03-17 22:37:44 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:37:44 utils.py:140] ==================================
I 03-17 22:38:04 utils.py:131] === Checking the job status... ===
I 03-17 22:38:05 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:38:05 utils.py:140] ==================================
I 03-17 22:38:25 utils.py:131] === Checking the job status... ===
I 03-17 22:38:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:38:26 utils.py:140] ==================================
I 03-17 22:38:46 utils.py:131] === Checking the job status... ===
I 03-17 22:38:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:38:47 utils.py:140] ==================================
I 03-17 22:39:07 utils.py:131] === Checking the job status... ===
I 03-17 22:39:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:39:09 utils.py:140] ==================================
I 03-17 22:39:29 utils.py:131] === Checking the job status... ===
I 03-17 22:39:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:39:30 utils.py:140] ==================================
I 03-17 22:39:51 utils.py:131] === Checking the job status... ===
I 03-17 22:39:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:39:51 utils.py:140] ==================================
I 03-17 22:40:12 utils.py:131] === Checking the job status... ===
I 03-17 22:40:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:40:12 utils.py:140] ==================================
I 03-17 22:40:32 utils.py:131] === Checking the job status... ===
I 03-17 22:40:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:40:34 utils.py:140] ==================================
I 03-17 22:40:54 utils.py:131] === Checking the job status... ===
I 03-17 22:40:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:40:55 utils.py:140] ==================================
I 03-17 22:41:15 utils.py:131] === Checking the job status... ===
I 03-17 22:41:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:41:16 utils.py:140] ==================================
I 03-17 22:41:36 utils.py:131] === Checking the job status... ===
I 03-17 22:41:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:41:37 utils.py:140] ==================================
I 03-17 22:41:57 utils.py:131] === Checking the job status... ===
I 03-17 22:42:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:42:01 utils.py:140] ==================================
I 03-17 22:42:21 utils.py:131] === Checking the job status... ===
I 03-17 22:42:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:42:22 utils.py:140] ==================================
I 03-17 22:42:43 utils.py:131] === Checking the job status... ===
I 03-17 22:42:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:42:46 utils.py:140] ==================================
I 03-17 22:43:06 utils.py:131] === Checking the job status... ===
I 03-17 22:43:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:43:07 utils.py:140] ==================================
I 03-17 22:43:27 utils.py:131] === Checking the job status... ===
I 03-17 22:43:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:43:27 utils.py:140] ==================================
I 03-17 22:43:47 utils.py:131] === Checking the job status... ===
I 03-17 22:43:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:43:48 utils.py:140] ==================================
I 03-17 22:44:08 utils.py:131] === Checking the job status... ===
I 03-17 22:44:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:44:09 utils.py:140] ==================================
I 03-17 22:44:29 utils.py:131] === Checking the job status... ===
I 03-17 22:44:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:44:30 utils.py:140] ==================================
I 03-17 22:44:50 utils.py:131] === Checking the job status... ===
I 03-17 22:44:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:44:51 utils.py:140] ==================================
I 03-17 22:45:11 utils.py:131] === Checking the job status... ===
I 03-17 22:45:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:45:12 utils.py:140] ==================================
I 03-17 22:45:32 utils.py:131] === Checking the job status... ===
I 03-17 22:45:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:45:33 utils.py:140] ==================================
I 03-17 22:45:53 utils.py:131] === Checking the job status... ===
I 03-17 22:45:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:45:54 utils.py:140] ==================================
I 03-17 22:46:14 utils.py:131] === Checking the job status... ===
I 03-17 22:46:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:46:15 utils.py:140] ==================================
I 03-17 22:46:35 utils.py:131] === Checking the job status... ===
I 03-17 22:46:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:46:36 utils.py:140] ==================================
I 03-17 22:46:56 utils.py:131] === Checking the job status... ===
I 03-17 22:46:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:46:56 utils.py:140] ==================================
I 03-17 22:47:17 utils.py:131] === Checking the job status... ===
I 03-17 22:47:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:47:17 utils.py:140] ==================================
I 03-17 22:47:38 utils.py:131] === Checking the job status... ===
I 03-17 22:47:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:47:38 utils.py:140] ==================================
I 03-17 22:47:59 utils.py:131] === Checking the job status... ===
I 03-17 22:47:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:47:59 utils.py:140] ==================================
I 03-17 22:48:19 utils.py:131] === Checking the job status... ===
I 03-17 22:48:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:48:20 utils.py:140] ==================================
I 03-17 22:48:40 utils.py:131] === Checking the job status... ===
I 03-17 22:48:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:48:41 utils.py:140] ==================================
I 03-17 22:49:01 utils.py:131] === Checking the job status... ===
I 03-17 22:49:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:49:02 utils.py:140] ==================================
I 03-17 22:49:22 utils.py:131] === Checking the job status... ===
I 03-17 22:49:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:49:23 utils.py:140] ==================================
I 03-17 22:49:43 utils.py:131] === Checking the job status... ===
I 03-17 22:49:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:49:44 utils.py:140] ==================================
I 03-17 22:50:04 utils.py:131] === Checking the job status... ===
I 03-17 22:50:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:50:05 utils.py:140] ==================================
I 03-17 22:50:25 utils.py:131] === Checking the job status... ===
I 03-17 22:50:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:50:26 utils.py:140] ==================================
I 03-17 22:50:46 utils.py:131] === Checking the job status... ===
I 03-17 22:50:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:50:46 utils.py:140] ==================================
I 03-17 22:51:07 utils.py:131] === Checking the job status... ===
I 03-17 22:51:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:51:07 utils.py:140] ==================================
I 03-17 22:51:28 utils.py:131] === Checking the job status... ===
I 03-17 22:51:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:51:28 utils.py:140] ==================================
I 03-17 22:51:48 utils.py:131] === Checking the job status... ===
I 03-17 22:51:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:51:49 utils.py:140] ==================================
I 03-17 22:52:09 utils.py:131] === Checking the job status... ===
I 03-17 22:52:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:52:10 utils.py:140] ==================================
I 03-17 22:52:30 utils.py:131] === Checking the job status... ===
I 03-17 22:52:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:52:31 utils.py:140] ==================================
I 03-17 22:52:51 utils.py:131] === Checking the job status... ===
I 03-17 22:52:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:52:52 utils.py:140] ==================================
I 03-17 22:53:12 utils.py:131] === Checking the job status... ===
I 03-17 22:53:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:53:13 utils.py:140] ==================================
I 03-17 22:53:33 utils.py:131] === Checking the job status... ===
I 03-17 22:53:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:53:34 utils.py:140] ==================================
I 03-17 22:53:54 utils.py:131] === Checking the job status... ===
I 03-17 22:53:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:53:55 utils.py:140] ==================================
I 03-17 22:54:15 utils.py:131] === Checking the job status... ===
I 03-17 22:54:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:16 utils.py:140] ==================================
I 03-17 22:54:36 utils.py:131] === Checking the job status... ===
I 03-17 22:54:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:37 utils.py:140] ==================================
I 03-17 22:54:57 utils.py:131] === Checking the job status... ===
I 03-17 22:54:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:57 utils.py:140] ==================================
I 03-17 22:55:18 utils.py:131] === Checking the job status... ===
I 03-17 22:55:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:18 utils.py:140] ==================================
I 03-17 22:55:39 utils.py:131] === Checking the job status... ===
I 03-17 22:55:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:39 utils.py:140] ==================================
I 03-17 22:55:59 utils.py:131] === Checking the job status... ===
I 03-17 22:56:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:00 utils.py:140] ==================================
I 03-17 22:56:20 utils.py:131] === Checking the job status... ===
I 03-17 22:56:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:21 utils.py:140] ==================================
I 03-17 22:56:41 utils.py:131] === Checking the job status... ===
I 03-17 22:56:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:42 utils.py:140] ==================================
I 03-17 22:57:02 utils.py:131] === Checking the job status... ===
I 03-17 22:57:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:03 utils.py:140] ==================================
I 03-17 22:57:23 utils.py:131] === Checking the job status... ===
I 03-17 22:57:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:24 utils.py:140] ==================================
I 03-17 22:57:44 utils.py:131] === Checking the job status... ===
I 03-17 22:57:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:45 utils.py:140] ==================================
I 03-17 22:58:05 utils.py:131] === Checking the job status... ===
I 03-17 22:58:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:06 utils.py:140] ==================================
I 03-17 22:58:26 utils.py:131] === Checking the job status... ===
I 03-17 22:58:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:26 utils.py:140] ==================================
I 03-17 22:58:47 utils.py:131] === Checking the job status... ===
I 03-17 22:58:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:47 utils.py:140] ==================================
I 03-17 22:59:07 utils.py:131] === Checking the job status... ===
I 03-17 22:59:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:08 utils.py:140] ==================================
I 03-17 22:59:28 utils.py:131] === Checking the job status... ===
I 03-17 22:59:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:29 utils.py:140] ==================================
I 03-17 22:59:49 utils.py:131] === Checking the job status... ===
I 03-17 22:59:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:50 utils.py:140] ==================================
I 03-17 23:00:10 utils.py:131] === Checking the job status... ===
I 03-17 23:00:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:11 utils.py:140] ==================================
I 03-17 23:00:31 utils.py:131] === Checking the job status... ===
I 03-17 23:00:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:32 utils.py:140] ==================================
I 03-17 23:00:52 utils.py:131] === Checking the job status... ===
I 03-17 23:00:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:53 utils.py:140] ==================================
I 03-17 23:01:13 utils.py:131] === Checking the job status... ===
I 03-17 23:01:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:14 utils.py:140] ==================================
I 03-17 23:01:34 utils.py:131] === Checking the job status... ===
I 03-17 23:01:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:35 utils.py:140] ==================================
I 03-17 23:01:55 utils.py:131] === Checking the job status... ===
I 03-17 23:01:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:55 utils.py:140] ==================================
I 03-17 23:02:16 utils.py:131] === Checking the job status... ===
I 03-17 23:02:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:16 utils.py:140] ==================================
I 03-17 23:02:36 utils.py:131] === Checking the job status... ===
I 03-17 23:02:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:37 utils.py:140] ==================================
I 03-17 23:02:57 utils.py:131] === Checking the job status... ===
I 03-17 23:02:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:58 utils.py:140] ==================================
I 03-17 23:03:18 utils.py:131] === Checking the job status... ===
I 03-17 23:03:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:03:19 utils.py:140] ==================================
I 03-17 23:03:39 utils.py:131] === Checking the job status... ===
I 03-17 23:03:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:03:40 utils.py:140] ==================================
I 03-17 23:04:00 utils.py:131] === Checking the job status... ===
I 03-17 23:04:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:01 utils.py:140] ==================================
I 03-17 23:04:21 utils.py:131] === Checking the job status... ===
I 03-17 23:04:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:22 utils.py:140] ==================================
I 03-17 23:04:42 utils.py:131] === Checking the job status... ===
I 03-17 23:04:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:43 utils.py:140] ==================================
I 03-17 23:05:03 utils.py:131] === Checking the job status... ===
I 03-17 23:05:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:03 utils.py:140] ==================================
I 03-17 23:05:24 utils.py:131] === Checking the job status... ===
I 03-17 23:05:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:24 utils.py:140] ==================================
I 03-17 23:05:44 utils.py:131] === Checking the job status... ===
I 03-17 23:05:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:45 utils.py:140] ==================================
I 03-17 23:06:05 utils.py:131] === Checking the job status... ===
I 03-17 23:06:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:06 utils.py:140] ==================================
I 03-17 23:06:26 utils.py:131] === Checking the job status... ===
I 03-17 23:06:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:27 utils.py:140] ==================================
I 03-17 23:06:47 utils.py:131] === Checking the job status... ===
I 03-17 23:06:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:48 utils.py:140] ==================================
I 03-17 23:07:08 utils.py:131] === Checking the job status... ===
I 03-17 23:07:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:09 utils.py:140] ==================================
I 03-17 23:07:29 utils.py:131] === Checking the job status... ===
I 03-17 23:07:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:30 utils.py:140] ==================================
I 03-17 23:07:50 utils.py:131] === Checking the job status... ===
I 03-17 23:07:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:51 utils.py:140] ==================================
I 03-17 23:08:11 utils.py:131] === Checking the job status... ===
I 03-17 23:08:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:12 utils.py:140] ==================================
I 03-17 23:08:32 utils.py:131] === Checking the job status... ===
I 03-17 23:08:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:32 utils.py:140] ==================================
I 03-17 23:08:52 utils.py:131] === Checking the job status... ===
I 03-17 23:08:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:53 utils.py:140] ==================================
I 03-17 23:09:13 utils.py:131] === Checking the job status... ===
I 03-17 23:09:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:14 utils.py:140] ==================================
I 03-17 23:09:34 utils.py:131] === Checking the job status... ===
I 03-17 23:09:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:35 utils.py:140] ==================================
I 03-17 23:09:55 utils.py:131] === Checking the job status... ===
I 03-17 23:09:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:56 utils.py:140] ==================================
I 03-17 23:10:16 utils.py:131] === Checking the job status... ===
I 03-17 23:10:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:17 utils.py:140] ==================================
I 03-17 23:10:37 utils.py:131] === Checking the job status... ===
I 03-17 23:10:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:38 utils.py:140] ==================================
I 03-17 23:10:58 utils.py:131] === Checking the job status... ===
I 03-17 23:10:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:59 utils.py:140] ==================================
I 03-17 23:11:19 utils.py:131] === Checking the job status... ===
I 03-17 23:11:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:11:20 utils.py:140] ==================================
I 03-17 23:11:40 utils.py:131] === Checking the job status... ===
I 03-17 23:11:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:11:40 utils.py:140] ==================================
I 03-17 23:12:00 utils.py:131] === Checking the job status... ===
I 03-17 23:12:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:01 utils.py:140] ==================================
I 03-17 23:12:21 utils.py:131] === Checking the job status... ===
I 03-17 23:12:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:22 utils.py:140] ==================================
I 03-17 23:12:42 utils.py:131] === Checking the job status... ===
I 03-17 23:12:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:43 utils.py:140] ==================================
I 03-17 23:13:03 utils.py:131] === Checking the job status... ===
I 03-17 23:13:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:04 utils.py:140] ==================================
I 03-17 23:13:24 utils.py:131] === Checking the job status... ===
I 03-17 23:13:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:25 utils.py:140] ==================================
I 03-17 23:13:45 utils.py:131] === Checking the job status... ===
I 03-17 23:13:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:46 utils.py:140] ==================================
I 03-17 23:14:06 utils.py:131] === Checking the job status... ===
I 03-17 23:14:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:07 utils.py:140] ==================================
I 03-17 23:14:27 utils.py:131] === Checking the job status... ===
I 03-17 23:14:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:28 utils.py:140] ==================================
I 03-17 23:14:48 utils.py:131] === Checking the job status... ===
I 03-17 23:14:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:49 utils.py:140] ==================================
I 03-17 23:15:09 utils.py:131] === Checking the job status... ===
I 03-17 23:15:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:09 utils.py:140] ==================================
I 03-17 23:15:30 utils.py:131] === Checking the job status... ===
I 03-17 23:15:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:30 utils.py:140] ==================================
I 03-17 23:15:50 utils.py:131] === Checking the job status... ===
I 03-17 23:15:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:51 utils.py:140] ==================================
I 03-17 23:16:11 utils.py:131] === Checking the job status... ===
I 03-17 23:16:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:12 utils.py:140] ==================================
I 03-17 23:16:32 utils.py:131] === Checking the job status... ===
I 03-17 23:16:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:33 utils.py:140] ==================================
I 03-17 23:16:53 utils.py:131] === Checking the job status... ===
I 03-17 23:16:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:54 utils.py:140] ==================================
I 03-17 23:17:14 utils.py:131] === Checking the job status... ===
I 03-17 23:17:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:15 utils.py:140] ==================================
I 03-17 23:17:35 utils.py:131] === Checking the job status... ===
I 03-17 23:17:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:36 utils.py:140] ==================================
I 03-17 23:17:56 utils.py:131] === Checking the job status... ===
I 03-17 23:17:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:57 utils.py:140] ==================================
I 03-17 23:18:17 utils.py:131] === Checking the job status... ===
I 03-17 23:18:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:17 utils.py:140] ==================================
I 03-17 23:18:38 utils.py:131] === Checking the job status... ===
I 03-17 23:18:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:38 utils.py:140] ==================================
I 03-17 23:18:59 utils.py:131] === Checking the job status... ===
I 03-17 23:18:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:59 utils.py:140] ==================================
I 03-17 23:19:19 utils.py:131] === Checking the job status... ===
I 03-17 23:19:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:20 utils.py:140] ==================================
I 03-17 23:19:40 utils.py:131] === Checking the job status... ===
I 03-17 23:19:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:41 utils.py:140] ==================================
I 03-17 23:20:01 utils.py:131] === Checking the job status... ===
I 03-17 23:20:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:02 utils.py:140] ==================================
I 03-17 23:20:22 utils.py:131] === Checking the job status... ===
I 03-17 23:20:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:23 utils.py:140] ==================================
I 03-17 23:20:43 utils.py:131] === Checking the job status... ===
I 03-17 23:20:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:44 utils.py:140] ==================================
I 03-17 23:21:04 utils.py:131] === Checking the job status... ===
I 03-17 23:21:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:05 utils.py:140] ==================================
I 03-17 23:21:25 utils.py:131] === Checking the job status... ===
I 03-17 23:21:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:26 utils.py:140] ==================================
I 03-17 23:21:46 utils.py:131] === Checking the job status... ===
I 03-17 23:21:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:47 utils.py:140] ==================================
I 03-17 23:22:07 utils.py:131] === Checking the job status... ===
I 03-17 23:22:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:08 utils.py:140] ==================================
I 03-17 23:22:28 utils.py:131] === Checking the job status... ===
I 03-17 23:22:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:28 utils.py:140] ==================================
I 03-17 23:22:49 utils.py:131] === Checking the job status... ===
I 03-17 23:22:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:49 utils.py:140] ==================================
I 03-17 23:23:09 utils.py:131] === Checking the job status... ===
I 03-17 23:23:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:23:10 utils.py:140] ==================================
I 03-17 23:23:30 utils.py:131] === Checking the job status... ===
I 03-17 23:23:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:23:31 utils.py:140] ==================================
I 03-17 23:23:51 utils.py:131] === Checking the job status... ===
I 03-17 23:23:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:23:52 utils.py:140] ==================================
I 03-17 23:24:12 utils.py:131] === Checking the job status... ===
I 03-17 23:24:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:13 utils.py:140] ==================================
I 03-17 23:24:33 utils.py:131] === Checking the job status... ===
I 03-17 23:24:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:34 utils.py:140] ==================================
I 03-17 23:24:54 utils.py:131] === Checking the job status... ===
I 03-17 23:24:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:55 utils.py:140] ==================================
I 03-17 23:25:15 utils.py:131] === Checking the job status... ===
I 03-17 23:25:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:16 utils.py:140] ==================================
I 03-17 23:25:36 utils.py:131] === Checking the job status... ===
I 03-17 23:25:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:37 utils.py:140] ==================================
I 03-17 23:25:57 utils.py:131] === Checking the job status... ===
I 03-17 23:25:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:58 utils.py:140] ==================================
I 03-17 23:26:18 utils.py:131] === Checking the job status... ===
I 03-17 23:26:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:18 utils.py:140] ==================================
I 03-17 23:26:38 utils.py:131] === Checking the job status... ===
I 03-17 23:26:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:39 utils.py:140] ==================================
I 03-17 23:26:59 utils.py:131] === Checking the job status... ===
I 03-17 23:27:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:00 utils.py:140] ==================================
I 03-17 23:27:20 utils.py:131] === Checking the job status... ===
I 03-17 23:27:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:21 utils.py:140] ==================================
I 03-17 23:27:41 utils.py:131] === Checking the job status... ===
I 03-17 23:27:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:42 utils.py:140] ==================================
I 03-17 23:28:02 utils.py:131] === Checking the job status... ===
I 03-17 23:28:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:03 utils.py:140] ==================================
I 03-17 23:28:23 utils.py:131] === Checking the job status... ===
I 03-17 23:28:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:24 utils.py:140] ==================================
I 03-17 23:28:44 utils.py:131] === Checking the job status... ===
I 03-17 23:28:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:45 utils.py:140] ==================================
I 03-17 23:29:05 utils.py:131] === Checking the job status... ===
I 03-17 23:29:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:05 utils.py:140] ==================================
I 03-17 23:29:26 utils.py:131] === Checking the job status... ===
I 03-17 23:29:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:26 utils.py:140] ==================================
I 03-17 23:29:46 utils.py:131] === Checking the job status... ===
I 03-17 23:29:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:47 utils.py:140] ==================================
I 03-17 23:30:07 utils.py:131] === Checking the job status... ===
I 03-17 23:30:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:30:08 utils.py:140] ==================================
I 03-17 23:30:28 utils.py:131] === Checking the job status... ===
I 03-17 23:30:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:30:29 utils.py:140] ==================================
I 03-17 23:30:49 utils.py:131] === Checking the job status... ===
I 03-17 23:30:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:30:50 utils.py:140] ==================================
I 03-17 23:31:10 utils.py:131] === Checking the job status... ===
I 03-17 23:31:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:11 utils.py:140] ==================================
I 03-17 23:31:31 utils.py:131] === Checking the job status... ===
I 03-17 23:31:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:32 utils.py:140] ==================================
I 03-17 23:31:52 utils.py:131] === Checking the job status... ===
I 03-17 23:31:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:53 utils.py:140] ==================================
I 03-17 23:32:13 utils.py:131] === Checking the job status... ===
I 03-17 23:32:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:13 utils.py:140] ==================================
I 03-17 23:32:34 utils.py:131] === Checking the job status... ===
I 03-17 23:32:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:34 utils.py:140] ==================================
I 03-17 23:32:54 utils.py:131] === Checking the job status... ===
I 03-17 23:32:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:55 utils.py:140] ==================================
I 03-17 23:33:15 utils.py:131] === Checking the job status... ===
I 03-17 23:33:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:16 utils.py:140] ==================================
I 03-17 23:33:36 utils.py:131] === Checking the job status... ===
I 03-17 23:33:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:37 utils.py:140] ==================================
I 03-17 23:33:57 utils.py:131] === Checking the job status... ===
I 03-17 23:33:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:58 utils.py:140] ==================================
I 03-17 23:34:18 utils.py:131] === Checking the job status... ===
I 03-17 23:34:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:19 utils.py:140] ==================================
I 03-17 23:34:39 utils.py:131] === Checking the job status... ===
I 03-17 23:34:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:40 utils.py:140] ==================================
I 03-17 23:35:00 utils.py:131] === Checking the job status... ===
I 03-17 23:35:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:01 utils.py:140] ==================================
I 03-17 23:35:21 utils.py:131] === Checking the job status... ===
I 03-17 23:35:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:21 utils.py:140] ==================================
I 03-17 23:35:42 utils.py:131] === Checking the job status... ===
I 03-17 23:35:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:42 utils.py:140] ==================================
I 03-17 23:36:02 utils.py:131] === Checking the job status... ===
I 03-17 23:36:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:36:03 utils.py:140] ==================================
I 03-17 23:36:23 utils.py:131] === Checking the job status... ===
I 03-17 23:36:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:36:24 utils.py:140] ==================================
I 03-17 23:36:44 utils.py:131] === Checking the job status... ===
I 03-17 23:36:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:36:45 utils.py:140] ==================================
I 03-17 23:37:05 utils.py:131] === Checking the job status... ===
I 03-17 23:37:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:37:06 utils.py:140] ==================================
I 03-17 23:37:26 utils.py:131] === Checking the job status... ===
I 03-17 23:37:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:37:27 utils.py:140] ==================================
I 03-17 23:37:47 utils.py:131] === Checking the job status... ===
I 03-17 23:37:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:37:48 utils.py:140] ==================================
I 03-17 23:38:08 utils.py:131] === Checking the job status... ===
I 03-17 23:38:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:38:09 utils.py:140] ==================================
I 03-17 23:38:29 utils.py:131] === Checking the job status... ===
I 03-17 23:38:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:38:29 utils.py:140] ==================================
I 03-17 23:38:50 utils.py:131] === Checking the job status... ===
I 03-17 23:38:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:38:50 utils.py:140] ==================================
I 03-17 23:39:10 utils.py:131] === Checking the job status... ===
I 03-17 23:39:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:39:11 utils.py:140] ==================================
I 03-17 23:39:31 utils.py:131] === Checking the job status... ===
I 03-17 23:39:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:39:32 utils.py:140] ==================================
I 03-17 23:39:52 utils.py:131] === Checking the job status... ===
I 03-17 23:39:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:39:53 utils.py:140] ==================================
I 03-17 23:40:13 utils.py:131] === Checking the job status... ===
I 03-17 23:40:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:40:14 utils.py:140] ==================================
I 03-17 23:40:34 utils.py:131] === Checking the job status... ===
I 03-17 23:40:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:40:35 utils.py:140] ==================================
I 03-17 23:40:55 utils.py:131] === Checking the job status... ===
I 03-17 23:40:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:40:56 utils.py:140] ==================================
I 03-17 23:41:16 utils.py:131] === Checking the job status... ===
I 03-17 23:41:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:41:16 utils.py:140] ==================================
I 03-17 23:41:37 utils.py:131] === Checking the job status... ===
I 03-17 23:41:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:41:37 utils.py:140] ==================================
I 03-17 23:41:57 utils.py:131] === Checking the job status... ===
I 03-17 23:41:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:41:58 utils.py:140] ==================================
I 03-17 23:42:18 utils.py:131] === Checking the job status... ===
I 03-17 23:42:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:42:19 utils.py:140] ==================================
I 03-17 23:42:39 utils.py:131] === Checking the job status... ===
I 03-17 23:42:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:42:40 utils.py:140] ==================================
I 03-17 23:43:00 utils.py:131] === Checking the job status... ===
I 03-17 23:43:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:43:01 utils.py:140] ==================================
I 03-17 23:43:21 utils.py:131] === Checking the job status... ===
I 03-17 23:43:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:43:22 utils.py:140] ==================================
I 03-17 23:43:42 utils.py:131] === Checking the job status... ===
I 03-17 23:43:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:43:43 utils.py:140] ==================================
I 03-17 23:44:03 utils.py:131] === Checking the job status... ===
I 03-17 23:44:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:44:04 utils.py:140] ==================================
I 03-17 23:44:24 utils.py:131] === Checking the job status... ===
I 03-17 23:44:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:44:24 utils.py:140] ==================================
I 03-17 23:44:45 utils.py:131] === Checking the job status... ===
I 03-17 23:44:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:44:45 utils.py:140] ==================================
I 03-17 23:45:05 utils.py:131] === Checking the job status... ===
I 03-17 23:45:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:45:06 utils.py:140] ==================================
I 03-17 23:45:26 utils.py:131] === Checking the job status... ===
I 03-17 23:45:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:45:27 utils.py:140] ==================================
I 03-17 23:45:47 utils.py:131] === Checking the job status... ===
I 03-17 23:45:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:45:48 utils.py:140] ==================================
I 03-17 23:46:08 utils.py:131] === Checking the job status... ===
I 03-17 23:46:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:46:09 utils.py:140] ==================================
